packages/browser-agent/README.md

```diff
@@ -1,27 +1,148 @@
-This is the main module that clients would import into their applications.
+[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
 
-It provides APIs to initialize the agent:
+# New Relic Browser Agent
 
+The New Relic Browser Agent instruments your website and provides observability into the performance and behavior of your application. This NPM Library is an ***in-progress*** implementation of the New Relic Browser Agent, accessible through `NPM`. Please see the [Differences](#differences) and [Features](#features) sections to compare how this library differs from the other offerings of the New Relic Browser Agent.
+
+## Installation
+
+### Using Package Managers
+
+Using [npm](https://npmjs.org):
+
+```bash
+npm install @newrelic/browser-agent
+```
+
+or [yarn](https://yarnpkg.com/)
+
+```bash
+yarn add @newrelic/browser-agent
+```
+
+### Directly in HTML/JS
+
+Using [unpkg](https://unpkg.com/)
+
+```html
+<script src="https://unpkg.com/@newrelic/browser-agent/bundled"></script>
 ```
-import nr from 'nr-browser-core'
 
-nr.setConfiguration({
-  jserrors: {
-    harvestTimeSeconds: 30
-  }
+## Usage Examples
+
+### Basic Setup
+
+```javascript
+<< App.js >>
+// initialize the agent as close to 
+// the top level of the application as possible
+import NR from '@newrelic/browser-agent'
+const options = {
+    // See 'Configuring your application'
+}
+NR.start(options).then(() => {
+    console.log("Browser Agent Initialized!")
 })
+```
+
+### Instrumenting a Micro Front End
 
-nr.init({
-  beaconUrl: 'localhost:8181',
-  licenseKey: 'abc123',
-  appId: 2
+The New Relic Browser Agent can maintain separate configuration scopes for each generated bundle.  Multiple bundles, like those used in certain micro front end patterns, can each report their own scoped data to separate New Relic applications.
```

Comment:
The New Relic Browser Agent can maintain separate configuration scopes by creating new instances. Micro front end patterns can rely on separate NR instances and can thus each report their own scoped data to separate New Relic applications.